coalib/results/Result.py

```diff
@@ -25,7 +26,7 @@
                    'aspect',
                    'additional_info',
                    'debug_msg')
-class Result:
+class Result(Comparables):
```

Comment:
Yeah, I just got to know the difference between the 2 ways of subclassing, the real one and the virtual one. I had actually referred to https://pymotw.com/3/abc/ which doesn't explicitly state the difference. My bad . :)